### ISSUE TITLE
Double quotes in file headers break CSV import

### ISSUE THREAD

**Issue by eito-fis** [centerofci/mathesar#572]
## Description
A CSV file with double quotes in its header names will cause the CSV import to break. This is because we directly insert the CSV header names into the SQL statement that creates the new, imported table.

## Expected behavior
Quotes should either be ignored or included in the column name. Regardless, the import should not fail.

## To Reproduce
Make a copy of any testing CSV, and add double quotes ~around~ inside the name of a column. Then, try to upload.

## Additional context
I think sqlalchemy should automatically escape column names? If this is the case, creating `Column` objects then just using their name or str representation should work.

**Comment by linzhou-zhong** [comment]
I'll fix this bug.

**Comment by linzhou-zhong** [comment]
[update]: 

I tried these 3 solutions above:

1. `list(map(lambda col: col.replace('"', ""), reader.fieldnames))`

2. `list(map(lambda col: col.strip('\"'), reader.fieldnames))`

3. `list(map(lambda col: eval(col), reader.fieldnames))`

All three solutions can handle our issue **Double quotes in file headers**. But **solution 2** and **solution 3** cannot deal with the specific corner case: `'" column"_one "'` where a double-quote was added in the middle of the header's name.

As result, I decide to choose **solution 1** as our first feasible solution.
